paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -269,8 +279,23 @@ internal class DefaultFlowController @Inject internal constructor(
             is PaymentSelection.Link,
             is PaymentSelection.New.LinkInline -> confirmLink(paymentSelection, state)
             is PaymentSelection.New,
-            is PaymentSelection.Saved,
             null -> confirmPaymentSelection(paymentSelection, state)
+            is PaymentSelection.Saved -> {
+                if (paymentSelection.paymentMethod.type == PaymentMethod.Type.SepaDebit &&
+                    viewModel.paymentSelection?.hasAcknowledgedSepaMandate == false
+                ) {
+                    // We're legally required to show the customer the SEPA mandate before every payment/setup.
+                    // In the edge case where the customer never opened the sheet, and thus never saw the mandate,
+                    // we present the mandate directly.
+                    sepaMandateActivityLauncher.launch(
+                        SepaMandateContract.Args(
+                            merchantName = state.config?.merchantDisplayName ?: ""
```

Comment:
This doesn't seem right, the activity is expecting a non null merchant name, what makes an empty string better here?